kubernetes/helm_charts/core/hawkeye-superset/values.j2

```diff
@@ -60,6 +56,10 @@ init:
     - "superset db upgrade && superset init && superset fab create-admin --username admin --firstname Superset --lastname Admin --email admin@superset.com --password admin"
 
 # Using the same nginx variables as this also needs to be private LB.
+service:
```

Comment:
Can it not be behind nginx private lb as a location instead of creating new LB?

Reply:
hmm.. It can be, but then we'll have to expose the Nginx private LB IP to all, and all services are openly accessible from there. So in this case, if somebody wants superset, we can just provide a unique ip